src/libraries/System.Private.CoreLib/src/Internal/Console.iOS.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Text;
+
+namespace Internal
+{
+    public static partial class Console
+    {
+        public static unsafe void Write(string s)
+        {
+            byte[] bytes = Encoding.Unicode.GetBytes(s);
```

Comment:
This should just pin the string, similar to how it is done here: https://github.com/dotnet/runtime/blob/8918adf2e5255920aed3c61f9b92c2a51eec0fe3/src/libraries/System.Console/src/System/ConsolePal.iOS.cs#L19-L22